tests/helpers/start-app.js

```diff
@@ -2,7 +2,7 @@ import Ember from 'ember';
 import Application from '../../app';
 import config from '../../config/environment';
 import registerGMapsAsyncHelpers from './ember-cli-g-maps/register-async-helpers';
-
+import setControllerProperty from './set-controller-property'; // jshint ignore:line
```

Comment:
We need another solution here. `set-controller-property` is a bad helper. We should do something more specific. Why do you need this?